src/Moneyman.Services/TransactionService.cs

```diff
@@ -69,23 +70,33 @@ public Transaction GetById(int id)
       return _transactionRepository.Get(id);
     }
 
-    public bool Create(Transaction trans)
+    public ApiResponse<int> Create(Transaction trans)
     {
+		
```

Comment:
I think instead we should be validating against TransactionDto model. This would mean that we consume the dto as our parameter and of the mapping here